Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -952,6 +952,7 @@ public void Dispose()
 
             CoClasses?.Clear();
             RemoveEventHandlers();
+            VBEEvents.Terminate();
```

Comment:
Why do we do this here in addition to doing it in `UnhookEvents`, which is calles in the shutdown code?

Reply:
Because that was where the original code had terminated the events. The `UnhookEvents` merely deregister the class from listening to events but `Terminate` shutdowns *all* event notifications from all consumers. I decided to leave them in each place in case there was a need for the class to unhook events but without shutting down everyone else's notifications.